MEAnalysis/python/samples_base.py

```diff
@@ -97,7 +97,8 @@ def get_files(fname):
     return lines
 
 def getSitePrefix(fn=""):
-    return "root://storage01.lcg.cscs.ch/pnfs/lcg.cscs.ch/cms/trivcat" + fn
+    #return "root://storage01.lcg.cscs.ch/pnfs/lcg.cscs.ch/cms/trivcat" + fn
+    return "root://xrootd-cms.infn.it/" + fn #DS
```

Comment:
We run all analysis codes directly on the samples on T2, I hesitate to put the global redirector here as it might be less reliable than the direct connection. I think we can just count 100% on T2.